src/models/org.ts

```diff
@@ -1,6 +1,9 @@
 import { Member, MemberDictionary } from './member';
 import { PendingInvite, PendingInviteDictionary } from './pending-invite';
 
+export interface OrgInvitePostPayload {
```

Comment:
This should be a model for ‘Invitation’ which has a postPayload function that returns the email but should also be used for the pendingInvitation GET. 

We should always try to mimic the database tables on the cloud as models.